R/metadata-extraction_imagery_dataset.R

```diff
@@ -21,19 +21,25 @@ extract_flight_speed = function(exif) {
   speed = median(speed, na.rm = TRUE) |> round(2)
 
   return(speed)
-
 }
 
 # Get a polygon of the mission
 # image_merge_distance: The horizontal distance between images below which they are merged into one
 # mission polygon. Keep only contiguous patches > min_contig_area.
 #' @export
-extract_mission_polygon = function(exif, image_merge_distance, min_contig_area = 1600) {
-
-  exif = sf::st_transform(exif, 3310)
-  ptbuff = sf::st_buffer(exif, image_merge_distance)
-  polybuff = sf::st_union(ptbuff)
-  poly = sf::st_buffer(polybuff, -image_merge_distance + 1) |> sf::st_cast("MULTIPOLYGON")
+extract_mission_polygon = function(exif, image_merge_distance, min_contig_area = 1600, boundary_method = "concaveman") {
```

Comment:
Note that #41 should be merged before this PR. This branch was rebased onto that one so it contains both sets of changes.